iphone/Classes/TiViewProxy.m

```diff
@@ -285,9 +290,33 @@ -(void)replaceAt:(id)args
 
 -(void)remove:(id)arg
 {
-	ENSURE_SINGLE_ARG(arg,TiViewProxy);
 	ENSURE_UI_THREAD_1_ARG(arg);
 
+	
+#ifdef HYPERLOOP
+	// ofbuscate this selector
+	static SEL nativeObjSel = nil;
+	if (nativeObjSel == nil) nativeObjSel = NSSelectorFromString([NSString stringWithFormat:@"%@%@", @"native", @"Object"]);
+	// obfuscate this class name
+	static Class hlClassName = nil;
+	if (hlClassName == nil) hlClassName = NSClassFromString([NSString stringWithFormat:@"%@%@", @"Hyperloop",@"Class"]);
+	if ([arg isKindOfClass:[UIView class]] || (hlClassName != nil && [arg isKindOfClass:hlClassName])) {
```

Comment:
Shouldn't we use `HyperloopPointer` here? And the `hlClassName` can't be nil, because the static variable is set in 302 if it wasn't not before. So the `hlClassName != nil` can be removed.